lib/krane/deploy_task.rb

```diff
@@ -332,15 +331,8 @@ def validate_globals(resources)
       end
       global_names = FormattedLogger.indent_four(global_names.join("\n"))
 
-      if @allow_globals
-        msg = "The ability for this task to deploy global resources will be removed in the next version,"\
-              " which will affect the following resources:"
-        msg += "\n#{global_names}"
-        @logger.summary.add_paragraph(ColorizedString.new(msg).yellow)
-      else
-        @logger.summary.add_paragraph(ColorizedString.new("Global resources:\n#{global_names}").yellow)
-        raise FatalDeploymentError, "This command is namespaced and cannot be used to deploy global resources."
-      end
+      @logger.summary.add_paragraph(ColorizedString.new("Global resources:\n#{global_names}").yellow)
+      raise FatalDeploymentError, "This command is namespaced and cannot be used to deploy global resources."
```

Comment:
You could argue it's outside the scope of this PR but I don't want this to get lost/forgotten. Can this error be clear what the alternative is? e.g. that there's a global deploy command?